### PR TITLE
support cypher parameter

### PR DESCRIPTION
**Pull request by czpmango**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -175,12 +196,41 @@ func executeConsoleCmd(c cli.Cli, cmd int, args []string) {
...
+		if len(args) != 1 {
+			return
+		}
+		res := reg.FindAllStringSubmatch(args[0], -1)
+		if len(res) != 1 || len(res[0]) != 3 {
+			return
+		}
+
+		tmp := make(ParameterMap)
+		param := "{\"" + res[0][1] + "\"" + ":" + res[0][2] + "}"
```
Comment (yixinglu) [inline main.go:214]: I have a question, at preset this pr only support following **one key format** parameter?

```json
{
  "name": "value"
}
```

Comment (czpmango) [reply main.go:214]: Just as previously reviewed, only int/float/ Boolean/string is currently supported.

File: main.go [hunk 2]
```diff
@@ -388,12 +449,87 @@ func validateFlags() {
...
+	} else if v, ok := any.(map[string]interface{}); ok {
+		nv, er := Map2Nmap(map[string]interface{}(v))
+		if er != nil {
+			err = er
+		}
+		value.MVal = nv
+	} else {
+		// unsupport other Value type, use this function carefully
+		err = fmt.Errorf("Do not support convert %T to nebula.Value", any)
+	}
```
Comment (Aiee) [inline main.go:522]: You could specify what value types are supported to be parameterized.

Comment (czpmango) [reply main.go:522]: ACK.

**Review by jievince — APPROVED** [review]
LGTM

**Pull request merged**